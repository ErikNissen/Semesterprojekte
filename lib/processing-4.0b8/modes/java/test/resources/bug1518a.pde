version https://git-lfs.github.com/spec/v1
oid sha256:af81b41cc697e112da3c812e3ac7d3b7508dd07f15770730c3fd7ea84e6bba1e
size 286
