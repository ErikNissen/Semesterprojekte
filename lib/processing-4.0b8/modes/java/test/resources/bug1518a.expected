version https://git-lfs.github.com/spec/v1
oid sha256:4c1049c76ba6f8681545b9744419fe30d91a0897d0a24b1ce066fd708526d74a
size 949
